homeassistant/components/miflora/manifest.json

```diff
@@ -3,6 +3,7 @@
   "name": "Miflora",
   "documentation": "https://www.home-assistant.io/components/miflora",
   "requirements": [
+    "bluepy==1.1.4",
```

Comment:
miflora already depends on bluepy, so it will be installed when miflora is installed.

Reply:
So this change should not be necessary?

Reply:
Nope. The library itself does not define that dependency, as it can be used with different Bluetooth backends. Bluepy is the most reliable one, so I would use that for HASS. So that is the reason we need to define the dependency here.

Reply:
As you see from the linked bug: this change fixes some of the problems with miflora...

Reply:
Oh I see now that the `setup.py` and `requirements.txt` are out of sync for the lib… https://github.com/open-homeautomation/miflora/blob/master/requirements.txt
https://github.com/open-homeautomation/miflora/blob/master/setup.py